src/Form/UnregistrationAdherentReasonsChoiceType.php

```diff
@@ -9,9 +9,9 @@
 use Symfony\Component\Form\FormEvents;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class UnregistrationReasonsChoiceType extends AbstractType
+class UnregistrationAdherentReasonsChoiceType extends AbstractType
```

Comment:
this form can be used by two formTypes `UnregistrationAdherentType` & `UnregistrationUserType`, for me not need to rename it, no?